packages/vats/src/priceAuthorityRegistry.js

```diff
@@ -6,9 +6,9 @@ import {
   provideDurableMapStore,
 } from '@agoric/vat-data';
 import { provideLazy } from '@agoric/store';
-import { E } from '@endo/eventual-send';
+import { E } from '@endo/far';
 import { Far } from '@endo/marshal';
-import { PriceAuthorityI } from '../src/contractSupport/priceAuthority.js';
+import { PriceAuthorityI } from '@agoric/zoe/src/contractSupport/priceAuthority.js';
```

Comment:
So the registry moves but not the priceAuthority API itself? OK.

It has always seemed strange to me that [Price Authority docs](https://docs.agoric.com/guides/zoe/price-authority.html) are under zoe, but whatever.

Reply:
The plan for deploying this change to production is the general "upgrade all vats" I suppose (#8104 -ish)